### DISCUSSION TITLE
MissionControl 0.12.0

### DISCUSSION THREAD

**Discussion by ndeadly** [ndeadly/MissionControl#873] [General]
<p align="left">
<img alt="GitHub Releases" src="https://img.shields.io/github/downloads/ndeadly/MissionControl/v0.12.0/total">
<a href="https://discord.gg/gegfNZ5Ucz"><img alt="Discord Server" src="https://discordapp.com/api/guilds/905069757361971303/widget.png?style=shield"/></a>
</p>

# ❗****Important: This release requires Atmosphere version 1.8.0 or later****❗

This is the twenty-fifth official release of Mission Control.

This release brings full support for firmware 19.0.0, an improved rumble decoding implementation, and fixes a couple of long-standing issues affecting several titles. See below for the full changelog.

- HD rumble decoding was completely rewritten after the outgoing reports were found to have multiple additional packing formats not previously documented
  - All reports should now be decoded correctly, resulting in all-round more accurate vibrations
  - The changes also resolve an issue where controllers would vibrate randomly at inappropriate times in certain titles (eg. in menus) due to incorrect decoding of some reports

- Basic NFC emulation has been implemented in order to correctly respond to NFC requests with a "not ready" status.
  - This resolves issues where games attempting to query the controller for amiibo data would display error messages or even lock up.

New users landing on this release page should first check out the readme on the [main project page](https://github.com/ndeadly/MissionControl#readme) for the official project documentation. There you can find installation and usage instructions along with an FAQ section that will answer most of your questions.

### Changelog:
- Added support for firmware 19.0.0
- Improved accuracy of HD rumble decoding
- Fixed issue where controllers would vibrate erratically in certain titles
- Fixed issue where certain titles would display an error when attempting to query non-native controllers for amiibo data

<hr /><em>This discussion was created from the release <a href='https://github.com/ndeadly/MissionControl/releases/tag/v0.12.0'>MissionControl 0.12.0</a>.</em>

**Comment by lightplusing** [comment]
mission control-0 . 12 . 0-master-6e 41 a 4d ，If I connect the XBOX（one s and Elite Wireless Controller Series 2 is also ） controller and hibernate, the host will not wake up, and I need to restart it forcibly.

**Reply by lightplusing** [reply]
NS19.0.0-1.8.0 atmosphere

**Reply by ndeadly** [reply]
Do you run sys-clk? The wake from sleep thing is a known issue on latest firmware and it needs to be updated. I'm not aware of anything that would cause similar behaviour with mission control

**Reply by lightplusing** [reply]
yes,I had run the sys-clk, it means I need to wait the atmosphere upgrade?

**Reply by lightplusing** [reply]
If I turn off the power of the controller first and then sleep on the host, I can wake up the host normally

**Reply by ndeadly** [reply]
You need to update sys-clk if you haven't already

**Reply by lightplusing** [reply]
It's indeed sys-clk problem. After I updated it, I was able to wake it up normally.Thank you

**Reply by Jonathan6182** [reply]
Bro me ayudas?? Quiero poner el sistem control, a la última versión,  me respondes estaré activo

**Reply by IsaelcuevasMendez** [reply]
> Bro me ayudas?? Quiero poner el sistem control, a la última versión, me respondes estaré activo

¿Tienes la última version de atmosphere?

**Comment by lightplusing** [comment]
Can run on 19.0.1？

**Reply by ndeadly** [reply]
Yes

**Reply by M4DCORP** [reply]
hi i accidentally updated firmware 19.0.1, switch is now running again but i cant seem to get dualsense edge to connect anymore after uninstalling and reinstalling missioncontrol and rebooting

**Reply by M4DCORP** [reply]
nevermind, if anyone has this issua and is also stupid, turn off flight mode

**Reply by 45646sadas** [reply]
Firmware 19.0.1, after installing MissionControl and restarting, I seem unable to connect

**Comment by Wilson-varela30** [comment]
It doesn't work for dualshock4, I've been tried for 4 days :(

**Comment by zurdo360** [comment]
Someone while playing suddenly the control does not respond and the console goes crazy and has to be turned off after it happened to me 2 times the character goes crazy and the control no longer responds

**Reply by ndeadly** [reply]
Which controller/game?  What do you mean by goes crazy?

**Reply by lightplusing** [reply]
Yes, I've also encountered 19.0.1. It seems that if the handle is far away from the main engine, it will easily fail out of control. If it is close to the main engine, it will be normal and no problem.

**Reply by ndeadly** [reply]
This sounds like a signal integrity issue to be honest. Often happens when people have USB3 support enabled in atmosphere settings.ini, for example

**Comment by 45646sadas** [comment]
Firmware 19.0.1, after installing MissionControl and restarting, I seem unable to connect

**Comment by Dixter528** [comment]
Firmware 19.0.1, I can't connect my xbox one controller, I tried for like 3 hours and I couldn't connect it, can anyone help me? 
Firmware 19.0.1. No puedo conectar mi control de xbox one, lo intente 3 horas y no pude, alguien sabe si es un error o algo? 😞

**Comment by faza20010** [comment]
Hey, can I use it with 18.1.0 and Atmosphere 1.8.0, with a xbox one controller?

**Reply by ndeadly** [reply]
Yes, it will work with those versions. Xbox One controllers are supported, but only on the old firmware < 5.x.x.x. Later firmware changes the controller to use Bluetooth LE, which isn't currently supported publicly. If you're on the newer firmware you'll either need to downgrade your controller firmware, or you can try a test build from my discord server that has experimental support for Bluetooth LE.

**Reply by faza20010** [reply]
Hi, so it crashed, I already fix it by removing mission control, but i would like to know if that's because of my controller?

**Reply by ndeadly** [reply]
It's unlikely any crash is due to your specific controller. The cause depends on the nature of the crash. If the console crashes during boot you probably have mismatched Atmosphere/Mission Control versions. If it crashes after connecting the downgraded controller with the release version, you might have corrupted files in `/config/MissionControl/controllers`. If it's crashing with the BLE build from discord, yeah, that can happen. It's a work in progress and there are known issues.

**Comment by lightplusing** [comment]
Why does the Xbox elite  have to be re-matched every time? Ordinary XBox handle, you can press the key to link back. Only the xbox elite  needs to be re-matched every time?Elite  I have been downgraded to the original version through official website.

**Reply by ndeadly** [reply]
I think somebody complained about this once before. It's not a widespread issue, it's something about your specific controller. I have no idea what the cause is, maybe related to the controller firmware version. You could try clearing the pairing database on the console and see if it works afterwards.
